src/borg/repository.py

```diff
@@ -960,12 +960,11 @@ def replay_segments(self, index_transaction_id, segments_transaction_id):
             self.rollback()
 
     def get_storage_quota_use(self):
-        storage_quota_use = 0
-        for i, (segment, filename) in enumerate(self.io.segment_iterator()):
-            objects = list(self.io.iter_objects(segment))
-            for tag, key, offset, size, _ in objects:
-                storage_quota_use += header_size(tag) + size
-        return storage_quota_use
+        hints_path = os.path.join(self.path, "hints.%d" % self.get_transaction_id())
+        integrity_data = self._read_integrity(self.get_transaction_id(), "hints")
```

Comment:
please only call get_transaction_id once.

Reply:
guess this method is not called any more?

Reply:
It isn't, I forgot it was there. I'm gonna remove it soon.